doc/advanced_configuration.rst

```diff
@@ -24,6 +24,56 @@ the sphinx configuration dictionary:
 
 Keep in mind that both lists have to be of the same length.
 
+.. _build_pattern:
+
+Building examples matching a pattern
+====================================
+
+By default, Sphinx-Gallery execute only examples beginning with ``plot``. However,
+if this naming convention does not suit your project, you can modify this pattern
+in your Sphinx ``conf.py``. For example:
+
+.. code-block:: python
+
+    sphinx_gallery_conf = {
+        'filename_pattern'  : '/plot_compute_'
+    }
+
+will build all examples starting with ``plot_compute_``. The key ``filename_pattern`` accepts
+regular expressions which will be matched with the full path of the example. This is the reason
+the leading ``'/'`` is required. Users are advised to use ``os.path.sep`` instead of ``'/'``
```

Comment:
I would use use `os.sep` here. Although it's very likely equivalent, `os.path.sep` is too close to `os.pathsep` and could lead to confusion.